src/main.ts

```diff
@@ -29,8 +32,17 @@ import { getCommentMessage } from "#/main/modules/get_comment_message";
 export const _main_ = async () => {
   // Verify correct environment and request context
   requireEvent();
-  requirePullNumber();
+  requirePullNumber(); // No needed, remove it ?? 
   const pr = await requirePr();
+  
+  // exit bot if no cc0 waiver
+  require_cc0(); 
+  
+  // max file allowed 100
+  require_max_file_number();
```

Comment:
This should be the very first check, since we specifically want it to short circuit in the case that things are too complicated for the bot to handle.